cmd/main.go

```diff
@@ -87,6 +88,11 @@ func main() {
 				return err
 			}
 
+			bolt, err := bolt.New(cfg)
```

Comment:
@jirevwe we should define `datastore.NewDB(cfg)`. It will do the if/else and return `datastore.DatabaseClient`

Reply:
i see a import cycle here, bolt -> datastore.Application, datastore -> bolt.New

Reply:
Hmm, let me check

Reply:
Yep, there's an import cycle. What do you think @Subomi?